plutus-core-spec/plutus-core-specification.tex

```diff
@@ -970,84 +971,85 @@ \section{Built in types, functions, and values}
-from a transaction can be spent as inputs to a later transaction.
-Users do not have an account containing currency, but instead have the
-right to spend outputs from previous transactions which have not yet
-been spent (UTXOs).
-
-To spend an output, a user has to prove that they have the right to do
-so.  In a simple case, an output might have been locked with a public
-key and a user would have to use the corresponding private key to
-unlock it.  However, more complicated scenarios are possible: for
-example, a specified set of users might have to act together to spend
-a particular output, or an output might not be spendable until a
-certain period of time has passed since its production.
-
-The process of verifying that an output can be spent is called
-\textit{validation}, and is performed by executing \textit{scripts},
-which in our case will be Plutus Core programs.
-
-
-Cardano uses a validation strategy which we call the \textit{Extended
-  UTXO model}.  A full specification of the Extended UTXO model is
-currently being prepared: in the meantime the fundamental ideas can be
-found in~\citep{Zahnentferner18-Chimeric} and
-\citep{Zahnentferner18-UTxO}.
-
-The basic idea is that when a transaction wishes $T$ to consume an
-unspent output $U$, it applies a script called the \textit{validator}
-to another script called the \textit{redeemer}\footnote{The Extended
-  UTXO model also involves another script called the \textit{data
-    script}, but for simplicity we won't discuss this here}; the
-validator also has access to certain parts of the blockchain state,
-such as the current block number (\textit{blocknum}) and the hash
-\textit{txhash} of the current transaction.  If the result of applying
-the validator to the redeemer is the Plutus Core \texttt{error} term
-then validation fails and the output cannot be spent; otherwise
-validation succeeds and the output can be spent.
-
-The job of the validator is to check whether the transaction $T$ is
-allowed to consume the output $U$, and the redeemer provides evidence
-that $U$ can in fact be spent.  For example, if a user $X$ wishes to
-spend an output $U$ then $X$ might provide a redeemer consisting of a
-signature obtained from $U$ with a private key belonging to $X$ and
-the validator would use $X$'s public key to check that the signature
-is valid.  We consider this example in more detail in the next
-section.
-
-\section{Example}
-
-\subfile{figures/ValidatorExample}
-
-%\section{Erasure}
-
-%TO WRITE
-
-%\subfile{figures/PlutusCoreErasureGrammar}
-
-%\subfile{figures/PlutusCoreErasureReduction}
-
-%\subfile{figures/PlutusCoreErasureTheorem}
-
-%\section{Example}
-
-%\subfile{figures/PlutusCoreExampleAgain}
+\section{Examples}
+The section contains some relatively simple examples of Plutus Core
+code.
+
+\paragraph{Integer subtraction.}
+The following program subtracts 11 from 55, returning 44.
+
+\begin{verbatim}
+  (program 1.0.0
```

Comment:
Is it deliberate that this one is a program and the others are terms? If so, I'd probably say "the remaining examples will be terms, since terms are always embedded into programs in the same manner" or something.